modules/light-clients/07-tendermint/types/client_state_test.go

```diff
@@ -201,6 +202,276 @@ func (suite *TendermintTestSuite) TestInitialize() {
+				merklePath, err := commitmenttypes.ApplyPrefix(suite.chainB.GetPrefix(), merklePath)
+				suite.Require().NoError(err)
+
+				path, err = suite.chainB.Codec.Marshal(&merklePath)
+				suite.Require().NoError(err)
+
+				proof, proofHeight = suite.chainB.QueryProof(key)
+
+				consensusState := testingpath.EndpointB.GetConsensusState(testingpath.EndpointB.GetClientState().GetLatestHeight()).(*types.ConsensusState)
+				value, err = suite.chainB.Codec.MarshalInterface(consensusState)
+				suite.Require().NoError(err)
+			},
+			expPass: true,
+		},
+		{
+			name: "successful Connection verification",
+			malleate: func() {
+				key := host.ConnectionKey(testingpath.EndpointB.ConnectionID)
+				merklePath := commitmenttypes.NewMerklePath(string(key))
+				merklePath, err := commitmenttypes.ApplyPrefix(suite.chainB.GetPrefix(), merklePath)
+				suite.Require().NoError(err)
+
+				path, err = suite.chainB.Codec.Marshal(&merklePath)
+				suite.Require().NoError(err)
+
+				proof, proofHeight = suite.chainB.QueryProof(key)
+
+				connection := testingpath.EndpointB.GetConnection()
+				value, err = suite.chainB.Codec.Marshal(&connection)
+				suite.Require().NoError(err)
+			},
+			expPass: true,
+		},
+		{
+			name: "successful Channel verification",
+			malleate: func() {
+				key := host.ChannelKey(testingpath.EndpointB.ChannelConfig.PortID, testingpath.EndpointB.ChannelID)
+				merklePath := commitmenttypes.NewMerklePath(string(key))
+				merklePath, err := commitmenttypes.ApplyPrefix(suite.chainB.GetPrefix(), merklePath)
+				suite.Require().NoError(err)
+
+				path, err = suite.chainB.Codec.Marshal(&merklePath)
+				suite.Require().NoError(err)
+
+				proof, proofHeight = suite.chainB.QueryProof(key)
+
+				channel := testingpath.EndpointB.GetChannel()
+				value, err = suite.chainB.Codec.Marshal(&channel)
+				suite.Require().NoError(err)
+			},
+			expPass: true,
+		},
+		{
+			"successful PacketCommitment verification", func() {
+				// send from chainB to chainA since we are proving chainB sent a packet
+				packet := channeltypes.NewPacket(ibctesting.MockPacketData, 1, testingpath.EndpointB.ChannelConfig.PortID, testingpath.EndpointB.ChannelID, testingpath.EndpointA.ChannelConfig.PortID, testingpath.EndpointA.ChannelID, clienttypes.NewHeight(0, 100), 0)
+				err := testingpath.EndpointB.SendPacket(packet)
+				suite.Require().NoError(err)
+
+				// make packet commitment proof
+				key := host.PacketCommitmentKey(packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
+				merklePath := commitmenttypes.NewMerklePath(string(key))
+				merklePath, err = commitmenttypes.ApplyPrefix(suite.chainB.GetPrefix(), merklePath)
+				suite.Require().NoError(err)
+
+				path, err = suite.chainB.Codec.Marshal(&merklePath)
+				suite.Require().NoError(err)
+
+				proof, proofHeight = testingpath.EndpointB.QueryProof(key)
+
+				value = channeltypes.CommitPacket(suite.chainA.App.GetIBCKeeper().Codec(), packet)
+			}, true,
```

Comment:
I think you missed adding `name`, `malleate` and `expPass` for this case. But curious how come you added for others? I think in most other testcases in ibc-go the field names of the anonymous struct aren't used.

fwiw, I feel totally indifferent about adding the field names or leaving them out. Both is fine imo! :)

Reply:
Copied/pasted the original tests which had the naming. I think I'll remove them. I also feel indifferent, but figure best to just stick with what we have rather than split up convention.